### DISCUSSION TITLE
Asus TUF F15 2023 has very limited settings? (Intel version)

### DISCUSSION THREAD

**Discussion by 47chromo** [seerge/g-helper#1007] [General]
Yeah so just wondering if the F15 2023 has limited settings because it's Intel or because it's relatively new. Have no idea how the PL1 + PL2 settings work so can't really lower my CPU temps while gaming. Would be neat if we got the temp limit feature that I've seen on other devices and an individual power limit setting.
![image](https://github.com/seerge/g-helper/assets/141475433/a00701a4-7de6-4899-83b5-1177007824c4)

**Comment by IceStormNG** [comment]
This is not supported on Intel as ASUS has no endpoints for that. I would recommend using Throttlestop for this as it can do all of that.

**Comment by seerge** [comment]
@47chromo PL1 (long time) and PL2 (short boost) are your power limits for CPU - you should be able to limit power with them (if you want to, but keep in mind that if you limit too much - your computer will be slow) :)  It's pretty much same as for any other device.

Temp limit feature, is only for AMD. And it works in a same way as power limit -> it will just (very badly) limit power of your CPU when it hits that temp.

**Reply by 47chromo** [reply]
Preciate it man. Is there a PL1 PL2 value you'd recommend to only get the CPU to like ~3.9ghz? Changing it to lower values still allows the CPU to fully boost for some reason.

**Reply by seerge** [reply]
Why do you want to lower your CPU frequencies in a first place ? I would recommend keeping default power limits (i.e. not clicking apply power at all), then it will be whatever asus has defined for your laptop for each mode.

**Reply by 47chromo** [reply]
Temps go up to 95 degrees Celsius which I'm not personally a fan of, and it makes the keyboard hot asf. Think I'll change the max boost frequency in reg editor.

**Reply by IceStormNG** [reply]
You can do that via throttlestop. If not, you have to remove CFG Lock and Overclocking lock in BIOS. And yes, reducing max turbo a little has great effects on temps with often the same performance. Intel's upper boost levels are ridiculous. They're just for winning benchmarks and power/performance ratios is completely nuts on those.

**Reply by seerge** [reply]
Temps go high from power consumed, not from frequencies :) if that bothers you try to limit power then to a certain amount, where you don't suffer too much, like 50-60w :)

**Reply by IceStormNG** [reply]
<img width="222" alt="image" src="https://github.com/seerge/g-helper/assets/12786283/12f5fe87-a69c-4004-b7d6-74888924c3dd">

FIVR is the option you look for. There it has the turbo ratios.

**Reply by 47chromo** [reply]
thank you guys for the replies I'll experiment with a low PL for now and if that fails will go for throttlestop

**Reply by IceStormNG** [reply]
> Temps go high from power consumed, not from frequencies :) if that bothers you try to limit power then to a certain amount, where you don't suffer too much, like 50-60w :)

There is this nasty little thing called "Voltage". and higher frequencies need higher voltages. The scale is not linear though, at some point you need a lot more voltage for just 100Mhz more clock. By limiting the clock you also cap the max voltage, which reduces power draw and therefore heat. reducing it by 200-300Mhz sometimes has a noticeably effect on thermals and power draw. This also means: more power for the GPU. And there is barely a need for those upper few 100Mhz. You have to find the value that your apps and games work well with. I also do that. And it does a lot actually. Was already the case on my 11th gen M16, and now also on the 2023. This might be different on AMD, but on Intel it is like this.

**Reply by 47chromo** [reply]
Could you share your throttlestop settings if you don't mind? (assuming that you also have an Intel CPU). Never tried undervolting and don't want to make any mistakes

**Reply by IceStormNG** [reply]
I guess yours has 13th gen Intel H series, then you cannot undervolt. You can limit the turbo ratio once you disabled CFG Lock and Overclocking lock in your BIOS. If there is no setting for that, you will have to set the setup variables via a modified grub loader, which is an advanced topic.

Then you can adjust these values here.
<img width="1128" alt="image" src="https://github.com/seerge/g-helper/assets/12786283/5fda2cd9-fc87-490d-8eb2-26d9380126fc">

You can also apply PL1 and PL2 as well as TAU in TPL settings.
<img width="736" alt="image" src="https://github.com/seerge/g-helper/assets/12786283/c8b442dd-aa0c-49e6-991c-e1fc3c23bf34">


Temp limit can be configured here:
<img width="962" alt="image" src="https://github.com/seerge/g-helper/assets/12786283/58e09d3b-25e6-4fd6-86de-a0b0bd81bda0">

PROCHOT offset is the reduction from 100°C. So 5 means 95°C thermal limit. 10 would be 90°C and so on. Intel's default is 0 = 100°C. ASUS default is 5 = 95°C. ASUS default is actually fine.
